### PR TITLE
Device: Forward Education Breakout Board + Relay + Pump v1.0

### PR DESCRIPTION
**Pull request by brian-inksmith**
This pull request will start the registration of your device in the Jacdac catalog.

### CODE REVIEW

**Comment by brian-inksmith** [comment]
@microsoft-github-policy-service agree [company="Forward Education"]

File: devices/forward-education/breakoutboardrelaypumpv10.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "forward-education-breakoutboardrelaypumpv10",
+  "name": "Breakout Board + Relay + Pump",
+  "company": "Forward Education",
+  "description": "0x3412171b\n",
```
Comment (pelikhan) [inline devices/forward-education/breakoutboardrelaypumpv10.json:5]: do you have a better description?

Comment (brian-inksmith) [reply devices/forward-education/breakoutboardrelaypumpv10.json:5]: Whoops!  That was only meant to be for the product identifier that I couldnt plop into the form.

The description (for now) would be:
`The Climate Action Kit Breakout Board from Forward Education.  Use this board with your micro:bit and your sensor and motor modules to learn to code and build prototype hardware.`

**Comment by pelikhan** [comment]
@brian-inksmith  did you use https://microsoft.github.io/jacdac-docs/tools/device-registration/ to generate this PR?

**Comment by brian-inksmith** [comment]
> @brian-inksmith did you use https://microsoft.github.io/jacdac-docs/tools/device-registration/ to generate this PR?

Yes I did.

**Pull request merged**